### PR TITLE
Cs/update/health check types

### PR DESCRIPTION
**Pull request by SmittieC**
## Description

Add ability to do liveness and readiness checks with commands also.

## Types of changes

_Put an `x` in the boxes that apply._

- [ ] Bugfix (`fix`): A non-breaking change which fixes an issue or bug
- [x] New Feature (`feat`): A non-breaking change which adds a new feature or new functionality
- [ ] Tests (`test`): Adding missing tests or correcting existing tests
- [ ] Refactoring (`refactor`): A code change that neither fixes a bug nor adds a feature, e.g Deleting unused or redundant code
- [ ] Documentation (`docs`): Documentation only changes
- [ ] Other (`style`, `build`, `ci`, `performance`): Formatting, linting or styling. Changes to the CI pipeline. Performance improvements. Changes that affect the build system.

## Further Information

JIRA Ticket: <https://uxi.atlassian.net/browse/BEP-XXX>

Any relevant logs, screenshots, testing output, etc...

### CODE REVIEW

File: charts/aruba-uxi/CHANGELOG.md [hunk 1]
```diff
@@ -18,6 +18,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/) and this p
 
 - what has been fixed
 
+## [2.4.2] - 2021-12-14
+
+### Added
+
+Ability to use commands to do liveness and readiness checks
```
Comment (deontpearson) [inline charts/aruba-uxi/CHANGELOG.md:25]: mind just adding an example of how to do it

Comment (SmittieC) [reply charts/aruba-uxi/CHANGELOG.md:25]: Sure

File: charts/aruba-uxi/values.example.yaml [hunk 2]
```diff
@@ -274,6 +274,14 @@ aruba-uxi:
...
       readinessProbe:
         path: /readyz
+      ## Enables the liveness probe to run the specified command (Optional)
+      ##
+      livenessProbe:
+        command: "ls -a"
+      ## Enables the readiness probe to run the specified command (Optional)
+      ##
+      readinessProbe:
+        command: "ls -a"
```
Comment (deontpearson) [inline charts/aruba-uxi/values.example.yaml:284]: this conflicts with the ones above. Given that workers will not have an http service exposed rather change the probes on line 270 and 275 to define the `command` and not `path` values

Comment (SmittieC) [reply charts/aruba-uxi/values.example.yaml:284]: Yeah I wasn't thinking here. I'll rather remove it and add an example in the changelog

File: charts/aruba-uxi/values.example.yaml [hunk 3]
```diff
@@ -267,13 +267,14 @@ aruba-uxi:
       ## Enables the liveness probe on the specified path (Optional)
       ## NB! for 'role:worker' this should be excluded
       ##
```
Comment (deontpearson) [inline charts/aruba-uxi/values.example.yaml:269]:
```suggestion
      ##
```

**Review by deontpearson — APPROVED** [review]

**Pull request merged**